src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7073,6 +7073,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 5 failures in `PowerShell-CI-linux`
Verify approved aliases list.All approved aliases present (no new aliases added, no aliases removed)
```powershell
Cannot bind argument to parameter 'ReferenceObject' because it is null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 547
547:         $result = Compare-Object -ReferenceObject $currentDisplayNameAliasList -DifferenceObject $aliasDisplayNameAliasList

```
Verify approved aliases list.All approved aliases have the correct 'AllScope' option
```powershell
Cannot bind argument to parameter 'ReferenceObject' because it is null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 559
559:         $result = Compare-Object -ReferenceObject $currentAllScopeOptionList -DifferenceObject  $aliasAllScopeOptionList

```
Verify approved aliases list.All approved aliases have the correct 'ReadOnly' option
```powershell
Cannot bind argument to parameter 'ReferenceObject' because it is null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 571
571:         $result = Compare-Object -ReferenceObject $currentReadOnlyOptionList -DifferenceObject  $aliasReadOnlyOptionList

```
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Cannot bind argument to parameter 'ReferenceObject' because it is null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 582
582:         $result = (Compare-Object -ReferenceObject $currentCmdletList.Name -DifferenceObject $cmdletList).InputObject

```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Cannot bind argument to parameter 'ReferenceObject' because it is null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 604
604:         $result = Compare-Object -ReferenceObject $currentCmdletList -DifferenceObject $CmdletList -Property ConfirmImpact -PassThru

```